### PR TITLE
just addressing some compiler warnings on windows

### PR DESCRIPTION
**Pull request by IamSanjid**
### For each PR
- [ ] Add Copyright if it missed:   
      - `"Copyright (c) 2019-present Axmol Engine contributors (see AUTHORS.md)."`
- [X] I have performed a self-review of my code.
       
   Optional:
   - [ ] I have checked readme and add important infos to this PR.
   - [ ] I have added/adapted some tests too.
          
### For core/new feature PR
- [ ] I have checked readme and add important infos to this PR.
- [ ] I have added thorough tests.

### CODE REVIEW

**Comment by rh101** [comment]
@IamSanjid It would be better if you do all the changes, then create the PR, but since you have already created the PR, then do all the changes and then commit in one go.  The reason is that now you've created the PR, every commit you make triggers the build system, or stops it and restarts it, and also sends out email notifications for every single commit to anyone who automatically subscribed to changes in this repo.

So, at the very least, please change this PR to a draft until it is ready to be reviewed.

Please be aware of this for any future changes.  It is always best to create the branch on your local PC, make any changes and commit them locally, and when it's complete, then create the pull request against the repository.

File: core/ui/UIWebView/UIWebViewImpl-win32.cpp [hunk 1]
```diff
@@ -200,17 +200,31 @@ inline std::string jsonParse(std::string_view s, std::string_view key, const int
     return "";
 }
 
+static LPWSTR to_lpwstr(std::string_view s)
```
Comment (halx99) [inline core/ui/UIWebView/UIWebViewImpl-win32.cpp]: Please use ntcvt for charset convert on windows, for example:

```cpp
auto ws = ntcvt::from_chars("hello");
auto as = ntcvt::from_chars(L"hello");
```

**Comment by IamSanjid** [comment]
> @IamSanjid It would be better if you do all the changes, then create the PR, but since you have already created the PR, then do all the changes and then commit in one go. The reason is that now you've created the PR, every commit you make triggers the build system, or stops it and restarts it, and then sends out email notifications for every single commit to anyone who automatically subscribed to changes in this repo.
> 
> So, at the very least, please change this PR to a draft until it is ready to be reviewed.
> 
> Please be aware of this for any future changes. It is always best to create the branch on your local PC, make any changes and commit them locally, and when it's complete, then create the pull request against the repository.

Ow I am sorry I didn't realize that

File: core/ui/UIWebView/UIWebViewImpl-win32.cpp (hunk 1 above)
Comment (IamSanjid) [reply core/ui/UIWebView/UIWebViewImpl-win32.cpp]: Done thank you!

**Comment by IamSanjid** [comment]
This should be it for now except regarding 3rd party, lua_bindings and unitialized member warning

**Pull request merged**